backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -754,6 +754,23 @@ public Project SetStatus(Project project, Workflow workflow)
                         }
                     });
                     break;
+                    case ("AP-EXE"): // Approve for ERP Exemption
```

Comment:
This looks identical to `case ("AP-ERP")`.  If it is, just allow them both to use the same code.
```
case ("AP-EXE"):
case ("AP-ERP"):
// code here
```